crates/rbuilder/src/building/builders/parallel_builder/groups.rs

```diff
@@ -31,15 +32,69 @@ struct GroupData {
     orders: Vec<SimulatedOrder>,
     reads: Vec<SlotKey>,
     writes: Vec<SlotKey>,
+    balance_reads: Vec<Address>,
+    balance_writes: Vec<Address>,
+    created_contracts: Vec<Address>,
+    destructed_contracts: Vec<Address>,
```

Comment:
these two can be simplified to code writes

Reply:
Thank you, combined the 2 under `code_writes` in the latest commit